File: 프로그래머스/lv1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.kt
class Solution {
    fun solution(price: Int, money: Int, count: Int): Long {
        var price = price.toLong()
        var firstPrice = price
        var money = money.toLong()

        repeat(count) {
            money -= price
            price += firstPrice
        }

        return if(money >= 0) 0 else -money
    }
}

File: 프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.swift
func solution(_ price: Int, _ money: Int, _ count: Int) -> Int64 {
    var answer = Int64(-money)
    
    for i in 1...count {
        answer += Int64(price) * Int64(i)
    }
    
    if answer > 0 {
        return Int64(answer)
    } else {
        return Int64(0)
    }
}